app/src/main/java/com/example/una/BugApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.example.una;
+
+import android.app.Application;
+
+import com.buglife.sdk.Buglife;
+import com.buglife.sdk.InvocationMethod;
+
+public class BugApplication extends Application {
+    @Override
+    public void onCreate() {
+        // Extended Application to use Buglife
+        super.onCreate();
+        Buglife.initWithEmail(this, getString(R.string.denize));
```

Comment:
where is this string? Also call it something suitable to the use case, `bug_report_email`?